torch_frame/nn/models/excelformer.py

```diff
@@ -37,30 +39,59 @@ def feature_mixup(
         y (Tensor): The target.
         num_classes (int): Number of classes.
         beta (float): The concentration parameter of the Beta distribution.
+            (default: :obj:`0.5`)
+        mixup_type (str): The mixup methods. `none` option means no mixup,
```

Comment:
```suggestion
        mixup_type (str, optional): The mixup methods. `none` option means no mixup,
```
Please make mixup_type optional instead.

Reply:
Sure, I have made `mixup_type` optional with supporting default `None` value mentioned above. The document has been adjusted accordingly.